client/notices_test.go

```diff
@@ -41,8 +42,10 @@ func (cs *clientSuite) TestNotice(c *C) {
 	}}`
 	notice, err := cs.cli.Notice("123")
 	c.Assert(err, IsNil)
+	thousand := uint32(1000)
```

Comment:
I'd prefer these called `uid` or `userID` or something (and `uid1` / `uid2` if there's more than one), to avoid the name being tied to the value.

Reply:
For testing the admin/non-admin default and filtering, I left them as `admin`, `nonAdmin`, and `otherNonAdmin`, but in all other cases I'm making these `uid[0-9]?`.

Reply:
Excellent, thanks.